openapi-specs/schema.json

```diff
@@ -4858,6 +4858,58 @@
             }
           }
         }
+      },
+      "delete": {
+        "description": "Cancel created backup with specified ID",
+        "operationId": "backups.cancel",
+        "x-serviceIds": [
+          "weaviate.local.backup"
+        ],
+        "tags": [
+          "backups"
+        ],
+        "parameters": [
+          {
+            "name": "backend",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Backup backend name e.g. filesystem, gcs, s3."
+          },
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The ID of a backup. Must be URL-safe and work as a filesystem path, only lowercase, numbers, underscore, minus characters allowed."
```

Comment:
Is there any place in the code where validate the backup ID ? I'm assuming we do it when we create a backup. Will it be a problem here if we use whichever user input ?